File: app/src/main/java/com/perflyst/twire/views/recyclerviews/auto_span_behaviours/EmoteAutoSpanBehaviour.java
package com.perflyst.twire.views.recyclerviews.auto_span_behaviours;

import android.content.Context;

import com.perflyst.twire.R;

/**
 * Created by Sebastian Rask on 09-05-2017.
 */

public class EmoteAutoSpanBehaviour implements AutoSpanBehaviour {
    @Override
    public String getElementSizeName() {
        return "";
    }

    @Override
    public int getElementWidth(Context context) {
        return (int) context.getResources().getDimension(R.dimen.chat_grid_emote_size);
    }
}

File: app/src/main/java/com/perflyst/twire/views/recyclerviews/auto_span_behaviours/EmoteAutoSpanBehaviour.kt
package com.perflyst.twire.views.recyclerviews.auto_span_behaviours

import android.content.Context
import com.perflyst.twire.R

/**
 * Created by Sebastian Rask on 09-05-2017.
 */
class EmoteAutoSpanBehaviour : AutoSpanBehaviour {
    override val elementSizeName: String get() = ""

    override fun getElementWidth(context: Context): Int {
        return context.resources.getDimension(R.dimen.chat_grid_emote_size).toInt()
    }
}
